cmd/export_ledger_entry_changes.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+// exportLedgerEntryChangesCmd represents the exportLedgerEntryChanges command
```

Comment:
This comment is a little repetitive, try to expand it to be more descriptive of actual function.

Reply:
Oh, that comment was autogenerated when I made the command. I'll remove it